README.md

```diff
@@ -37,6 +37,19 @@ fail. For instructions on deploying MLFlow you can start with [this
 guide](https://discourse.charmhub.io/t/deploying-charmed-mlflow-v2-and-kubeflow-to-eks/10973),
 ignoring the EKS specific steps.
 
+## Python and Tox Setup
```

Comment:
I think we want to move this as a subsection of [Running from a configured management environment using the driver](https://github.com/canonical/charmed-kubeflow-uats?tab=readme-ov-file#running-from-a-configured-management-environment-using-the-driver) because there are two ways of running UATs: directly from a Notebook or using the driver. So having this information there will make it clear.